src/ontology/foodon-edit.owl

```diff
@@ -22245,6 +22256,10 @@ SubClassOf(obo:FOODON_03305132 ObjectSomeValuesFrom(obo:RO_0001000 obo:FOODON_03
 
 # Class: obo:FOODON_03305135 (rice pudding)
 
+AnnotationAssertion(obo:IAO_0000115 obo:FOODON_03305135 "A pudding food product made from rice mixed with water or milk."@en)
+AnnotationAssertion(obo:IAO_0000119 obo:FOODON_03305135 "https://en.wikipedia.org/wiki/Rice_pudding"@en)
+AnnotationAssertion(schema:image obo:FOODON_03305135 "https://upload.wikimedia.org/wikipedia/commons/thumb/d/dc/Rice_pudding_vanilla_cinnamon.jpg/800px-Rice_pudding_vanilla_cinnamon.jpg?20070602064451 | https://upload.wikimedia.org/wikipedia/commons/thumb/4/46/Kheer.jpg/450px-Kheer.jpg | https://upload.wikimedia.org/wikipedia/commons/thumb/6/66/Arros_amb_llet.jpg/1280px-Arros_amb_llet.jpg"@en)
```

Comment:
I think it is sufficient to just have one low-res image; the main thing is to link off to Wikipedia for further imagery/ writeup.

Reply:
Thanks for that feedback, have amended in a later commit.